images/metering-ansible-operator/roles/meteringconfig/tasks/main.yml

```diff
@@ -1,14 +1,35 @@
 ---
 
-- name: Print meteringconfig values
+- name: run the new module
+  k8s_event:
+    state: present
+    name: test-https-emily109
+    namespace: ansible-emily
```

Comment:
`namespace: {{ meta.namespace }}`

Reply:
I threw the WIP on it. The main.yml is not being commited. For test only. Now reverted.

Reply:
I don't see a problem with committing this to main.yml for the time being. If you wanted to just test the event in the e2e suite, you could have run this locally.

Reply:
This wouldn't pass the e2e test either as the `ansible-emily` namespace doesn't exist in CI.